Docker/start-container.sh

```diff
@@ -3,14 +3,14 @@
 #Set user and group
 umask 0002
 PUID=${PUID:-`id -u squeezeboxserver`}
-PGID=${PGID:-`id -g squeezeboxserver`}
+PGID=${PGID:-`id -g nogroup`}
```

Comment:
The parameter to `id` must be the user, not the group. `id -g squeezeboxserver` would return the id for that user's group.

Reply:
Yes. That's what you have without being able to test it, argh...